kpatch-build/kpatch-build

```diff
@@ -65,6 +65,15 @@ LLD="${CROSS_COMPILE:-}ld.lld"
 READELF="${CROSS_COMPILE:-}readelf"
 OBJCOPY="${CROSS_COMPILE:-}objcopy"
 
+SUPPORTED_DISTRO="
+    fedora
+    rhel
+    centos
+    openEuler
+    photon
+    Anolis
```

Comment:
This should be lowercase: anolis

Reply:
This problme is fixed associated shell return value bug.